### PR TITLE
Feature/muwm 5075

### PR DESCRIPTION
**Pull request by kbui98**
Jira: https://jira.cac.washington.edu/browse/MUWM-5075

Researcher Resource List to compare links: https://docs.google.com/document/d/1aFd84m7gqJQ59g8HE6aTQC9pLMbabs_l_yfdNogDbFY/edit

### CODE REVIEW

File: myuw/data/resource_link_import.csv [hunk 1]
```diff
@@ -1,154 +1,168 @@
...
+Research,Research Administration,all,https://myresearch.washington.edu/,MyResearch Portal,,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/tools/sage/,System to Administer Grants Electronically (SAGE),,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/tools/,Research Administration Tools,,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/resources/funding-opportunities/,Funding Opportunities,,,,,,,no
+Research,"Research Centers, Shared Facilities, and Equipment",all,https://www.washington.edu/research/research-centers/,Research Centers and Institutes,,,,,,,no
+Research,"Research Centers, Shared Facilities, and Equipment",all,https://www.washington.edu/research/shared-research-facilities-resources/,Shared Research Facilities and Resources,,,,,,,no
+Research,"Research Centers, Shared Facilities, and Equipment",all,https://escience.washington.edu/,eScience Institute,,,,,,,no
+Research,Research Data and Publishing,all,https://www.lib.washington.edu/services/faculty,UW Libraries: Getting started with your research,,,,,,,no
```
Comment (wmwash) [inline myuw/data/resource_link_import.csv:159]: This title/page combo is a bit confusing to my eyes (may not be for a researcher). It appears to go to a generic UW Libraries page on how to conduct library research. Is that the intention? If so, we may want to specify it's about getting started with *library* research...

File: myuw/data/resource_link_import.csv [hunk 2]
```diff
@@ -1,154 +1,168 @@
...
+Teaching,Grading,all,https://gradepage.uw.edu,GradePage,,,,,,,no
+Teaching,Grading,all,https://apps.registrar.washington.edu/grade-change/pages/change.php,Change Submitted Grades,,,,,,,no
+Teaching,Grading,all,,,http://depts.washington.edu/grading,Faculty Resource on Grading,https://www.uwb.edu/registration/policies/grading,Grading Policies,https://www.tacoma.uw.edu/uwt/registrar/grading-policy,Grading Policy,no
+Teaching,Tools,all,https://canvas.uw.edu,Canvas LMS,,,,,,,no
+Teaching,Tools,all,https://catalyst.uw.edu/,Catalyst Web Tools,,,,,,,no
+Teaching,Tools,all,https://uw.hosted.panopto.com,Panopto Lecture Capture,,,,,,,no
+Teaching,Tools,all,https://itconnect.uw.edu/learn/tools/polleverywhere/,Poll Everywhere,,,,,,,no
+Teaching,Tools,all,https://coda.uw.edu,Course Stats,,,,,,,no
+Teaching,Tools,all,https://washington.zoom.us,Zoom,,,,,,,no
+Research,Research Administration,all,https://www.washington.edu/research/,UW Research Website,,,,,,,no
```
Comment (wmwash) [inline myuw/data/resource_link_import.csv:149]: We don't typically add "website" to site titles. I suggest it just be "UW Research"

File: myuw/data/resource_link_import.csv [hunk 3]
```diff
@@ -1,154 +1,168 @@
...
+Research,"Research Centers, Shared Facilities, and Equipment",all,https://escience.washington.edu/,eScience Institute,,,,,,,no
+Research,Research Data and Publishing,all,https://www.lib.washington.edu/services/faculty,UW Libraries: Getting started with your research,,,,,,,no
+Research,Research Data and Publishing,all,https://www.lib.washington.edu/dataservices/tools/services,Tools for Data-Related Research,,,,,,,no
+Research,Research Data and Publishing,all,https://guides.lib.uw.edu/research/dmg,Research Data Services,,,,,,,no
+Research,Research Data and Publishing,all,https://hsl.uw.edu/trail/,Translational Research and Information Lab (TRAIL),,,,,,,no
+Research,Research Data and Publishing,all,,,https://www.lib.washington.edu/openscholarship,"Open Scholarship Commons: Tools, Services, Consultations",https://guides.lib.uw.edu/bothell/digitalscholarship,Digital Scholarship,https://guides.lib.uw.edu/tacdigitalscholarship,Digital Scholarship,no
+Research,Research Data and Publishing,all,https://guides.lib.uw.edu/research/spoa,Support for Publishing,,,,,,,no
```
Comment (wmwash) [inline myuw/data/resource_link_import.csv:164]: Would "Scholarly Publishing and Open Access" be a more fitting title here?

**Review by wmwash — CHANGES_REQUESTED** [review]

File: myuw/data/resource_link_import.csv (hunk 1 above)
Comment (kbui98) [reply myuw/data/resource_link_import.csv]: Updated to "Getting started with library research" !

File: myuw/data/resource_link_import.csv (hunk 2 above)
Comment (kbui98) [reply myuw/data/resource_link_import.csv]: Updated to "UW Research" !

File: myuw/data/resource_link_import.csv (hunk 3 above)
Comment (kbui98) [reply myuw/data/resource_link_import.csv]: Yes I agree! Updated to "Scholarly Publishing and Open Access" ! Thanks :)

**Review by wmwash — APPROVED** [review]

**Review by fanglinfang — APPROVED** [review]

**Pull request merged**